pyproject.toml

```diff
@@ -10,11 +10,15 @@ keywords = ["Palantir", "Foundry", "SDK", "Client", "API"]
 packages = [{ include = "foundry" }]
 
 [tool.poetry.dependencies]
+annotated-types = ">=0.7.0"
+expects = ">=0.9.0"
+mockito = ">=1.5.1"
+pydantic = "^2.1.0"
+pytest = ">=7.4.0"
+pytest-asyncio = ">=0.23.0"
```

Comment:
test dependencies should be in their own group https://python-poetry.org/docs/managing-dependencies/ i.e., they aren't required to install and use the package

Reply:
will also need to update CI logic to install test deps